packages/grid/data-grid/src/tests/layout.DataGrid.test.tsx

```diff
@@ -486,6 +487,26 @@ describe('<DataGrid /> - Layout & Warnings', () => {
 
       expect(getByText('Size')).not.to.equal(null);
     });
+
+    it('should pass translations using the ThemeProvider', () => {
+      const theme = createMuiTheme({}, bgBG);
+
+      const { getByText } = render(
+        <div style={{ width: 300, height: 300 }}>
+          <ThemeProvider theme={theme}>
+            <DataGrid
+              {...baselineProps}
+              showToolbar
+              components={{
+                Toolbar: GridToolbar,
+              }}
+            />
+          </ThemeProvider>
+        </div>,
+      );
+
+      expect(getByText('Гъстота')).not.to.equal(null);
+    });
```

Comment:
I think that this test is too high-level. We don't need it.

Reply:
Consider this approach instead: https://github.com/mui-org/material-ui/blob/b5bce5007b6d025360fefa887eaf5b0206fab5ae/test/utils/describeConformanceV5.js#L40. We can copy the helper until we upgrade to v5 to be able to simply replace the import.

Reply:
I removed the test for now. If we don't need it it's better to avoid copying the helper.

Reply:
Ok, no preferences. I think that long term, we should have all the public components running inside describeConformance. Marija had opened https://github.com/mui-org/material-ui/issues/22371 related to this problem